libs/coin-framework/src/api/types.ts

```diff
@@ -6,15 +6,19 @@ export type BlockInfo = {
 
 export type Operation = {
   hash: string;
+  // TODO: check if useful
   address: string;
   type: string;
+  // TODO: check if we want Asset array
```

Comment:
lama-adapter match those values:
```
      case "IN"                              => OperationType.Receive
      case "OUT"                             => OperationType.Send
```

Reply:
How do you deal with cases where a tx does multiple things at once ?

for example mint (eg. you send/burn in order to receive another asset), or just more complex interactions basically

Can you link where this enum/type is defined ?

Reply:
It's defined here: https://github.com/LedgerHQ/lama-adapter/blob/main/modules/model/src/main/scala/lama/model/Operation.scala#L66

Reply:
done